exercises/matching-brackets/runtests.jl

```diff
@@ -0,0 +1,23 @@
+using Test
+
+include("matching-brackets.jl")
+
+@testset "Determine if brackets in expression are correctly matched" begin
+    @test matching_brackets("[]")::Bool == true  
+    @test matching_brackets("")::Bool == true
+    @test matching_brackets("[[")::Bool == false
+    @test matching_brackets("}{")::Bool == false
+    @test matching_brackets("{]")::Bool == false
+    @test matching_brackets("{ }")::Bool == true
+    @test matching_brackets("{[])")::Bool == false
+    @test matching_brackets("{[]}")::Bool == true
+    @test matching_brackets("{}[]")::Bool == true
+    @test matching_brackets("([{}({}[])])")::Bool == true
+    @test matching_brackets("{[)][]}")::Bool == false
+    @test matching_brackets("([{])")::Bool == false
+    @test matching_brackets("[({]})")::Bool == false
+    @test matching_brackets("{}[")::Bool == false
+    @test matching_brackets("[]]")::Bool == false
+    @test matching_brackets("(((185 + 223.85) * 15) - 543)/2")::Bool == true
+    @test matching_brackets("\\left(\\begin{array}{cc} \\frac{1}{3} & x\\\\ \\mathrm{e}^{x} &... x^2 \\end{array}\\right)")::Bool == true
```

Comment:
Please group similar tests together in testsets. There are a few ways to do it. The [canonical data](https://github.com/exercism/problem-specifications/blob/master/exercises/matching-brackets/canonical-data.json) has some suggestions.

For example you could have testsets for

- tests that contain one pair of brackets
- tests with nested brackets
- tests with brackets behind each other but not nested
- etc.

There's no right or wrong way but at least some grouping should be done here. That also enables the student to work on the exercise in a TDD-inspired way, by making their solution pass simple tests first and then expanding it to work on more complex cases.